tests/specs/tram-spec.js

```diff
@@ -310,25 +303,76 @@ const tests = (Tram) => describe('Tram', () => {
     it('should return a string', () => {
       const app = new Tram()
       app.addRoute('/404', errorPage)
-      expect(app.toString('/')).toEqual(stringify(errorPage()))
+      expect(app.toString('/')).toEqual(errorPage().outerHTML)
     })
   })
 
-  describe('html', () => {
+  describe('dom', () => {
+    const HTMLNS = 'http://www.w3.org/1999/xhtml'
+    const SVGNS = 'http://www.w3.org/2000/svg'
+
     it('should generate a dom tree', () => {
-      const tramTree = Tram.html()`<div><span></span></div>`
+      const tramTree = Tram.dom()`<div><span></span></div>`
       const docTree = document.createElement('div')
       docTree.appendChild(document.createElement('span'))
       expect(tramTree.outerHTML).toBe(docTree.outerHTML)
     })
 
     it('should take in a registry', () => {
-      const foo = () => Tram.html()`<div><span></span></div>`
-      const tramTree = Tram.html({foo})`<foo></foo>`
+      const foo = () => Tram.dom()`<div><span></span></div>`
```

Comment:
Why are we using foo for the test? Could this be named something more useful for the registry?

Reply:
good point, will fix!

Reply:
it has been done⚡️